docs/data-sources/resource.md

```diff
@@ -0,0 +1,61 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "twingate_resource Data Source - terraform-provider-twingate"
+subcategory: ""
+description: |-
+  Resources in Twingate represent servers on the private network that clients can connect to. Resources can be defined by IP, CIDR range, FQDN, or DNS zone. For more information, see the Twingate documentation https://docs.twingate.com/docs/resources-and-access-nodes.
+---
+
+# twingate_resource (Data Source)
+
+Resources in Twingate represent servers on the private network that clients can connect to. Resources can be defined by IP, CIDR range, FQDN, or DNS zone. For more information, see the Twingate [documentation](https://docs.twingate.com/docs/resources-and-access-nodes).
+
+## Example Usage
+
+```terraform
+data "twingate_resource" "foo" {
+  id = "<your resource's id>"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `id` (String) The ID of the Resource
+
+### Read-Only
+
+- `address` (String) The Resource's IP/CIDR or FQDN/DNS zone
+- `name` (String) The name of the Resource
+- `protocols` (List of Object) Restrict access to certain protocols and ports. By default or when this argument is not defined, there is no restriction, and all protocols and ports are allowed. (see [below for nested schema](#nestedatt--protocols))
```

Comment:
Change the following sentences to this:

`By default (when this argument is not defined) no restriction is applied, and all protocols and ports are allowed. See below for [nested schema](#nestedatt--protocols).`

Reply:
updated